node/lib/cmd/open.js

```diff
@@ -78,6 +78,7 @@ exports.configureParser = function (parser) {
 exports.executeableSubcommand = co.wrap(function *(args) {
     const colors = require("colors");
 
+    const DoWorkQueue      = require("../util/do_work_queue");
```

Comment:
Wait, no, unless I'm missing something, 71d4976b7fb52a421a661716899f06a78b10e689 is first commit and adds the `do_work_queue.js` files; 114291f76d3420095e146fc6e18e35bf181d5037 is second and uses them.